src/components/ftbot/TradeList.vue

```diff
@@ -204,6 +205,7 @@ watch(
       :scrollable="true"
       scroll-height="flex"
       @row-click="onRowClicked"
+      removableSort 
```

Comment:
how does this change relate to the actual PR? I don't see how this would change how charts are rendered?